Data/Sys/GameSettings/GL5.ini

```diff
@@ -17,3 +17,5 @@ EmulationIssues = Slow
 [ActionReplay]
 # Add action replay cheats here.
 
+[Video_Stereoscopy]
+StereoConvergence = 1
```

Comment:
Whole integers have worked well on all games I've tested. I don't know how useful it would be.